src/Team-Services-Bot.Api.UnitTests/Team-Services-Bot.Api.UnitTests.csproj

```diff
@@ -33,6 +33,7 @@
     <CodeAnalysisRuleSet>..\Common\TestCodeRules.ruleset</CodeAnalysisRuleSet>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <DocumentationFile>bin\Debug\Vsar.TSBot.UnitTests.xml</DocumentationFile>
+    <NoWarn>1591</NoWarn>
```

Comment:
Same issue - should be disabled for release configuration as well. Preferable this should be done in the code.